ReactiveCocoaTests/Swift/PropertySpec.swift

```diff
@@ -287,7 +350,23 @@ class PropertySpec: QuickSpec {
 				expect(values) == [ 0, 1, 2 ]
 			}
 
-			it("should complete when the underlying object deallocates") {
+			it("should yield a signal that sends the changes for the key path of the underlying object") {
+				var values: [Int] = []
+				property.signal.observeNext { value in
+					expect(value).notTo(beNil())
+					values.append((value as? Int) ?? -1)
```

Comment:
Only because you chose a magic number that happens to be different than the assertion, but that's a very fragile way of verifying that, and it doesn't correctly communicate what you *expect* in the test.